doc/advanced/devel/websocket-api.rst

```diff
@@ -235,7 +235,7 @@ for this purpose is `websocat`_.
 Example Architecture
 --------------------
 
-.. figure:: ../images/websocket-api/one-api-many-zeek.svg
+.. figure:: ../../images/websocket-api/one-api-many-zeek.svg
```

Comment:
```suggestion
.. figure:: /images/websocket-api/one-api-many-zeek.svg
```

I've used the absolute version elsewhere recently and it should work. I'd probably opt for that.